appleid.md

```diff
@@ -0,0 +1,27 @@
+# Apple ID
+Apple IDs are a requirement for Macbook support. 
+## Installation
+You will receive an invitation via email containing 
```

Comment:
Can you add a line return before lists, the formatting has gone a bit funky https://github.com/studio24/mac-setup/blob/94b12c0285d7e5f0ebc72cd0519f43421a206452/appleid.md

Reply:
Updated